File: app/src/main/java/edu/ucdenver/fallnotification/MainActivity.java
package edu.ucdenver.fallnotification;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.content.Intent;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onViewSessions(View view) {
        Intent intentViewSessions = new Intent(this, ViewSessions.class);
        startActivity(intentViewSessions);
    }

    public void onViewPatients(View view) {
        Intent intentViewPatients = new Intent(this, ViewPatients.class);
        startActivity(intentViewPatients);
    }

    public void onSetUpSession(View view) {
        Intent intentSetUpSession = new Intent(this, SetUpSession.class);
        startActivity(intentSetUpSession);
    }
}

File: app/src/main/java/edu/ucdenver/falldetection/MainActivity.kt
package edu.ucdenver.falldetection

import android.bluetooth.BluetoothAdapter
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.navigation.findNavController
import androidx.navigation.ui.AppBarConfiguration
import androidx.navigation.ui.setupActionBarWithNavController
import androidx.navigation.ui.setupWithNavController
import com.google.android.material.bottomnavigation.BottomNavigationView
import edu.ucdenver.falldetection.databinding.ActivityMainBinding


class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val navView: BottomNavigationView = binding.navView

        val navController = findNavController(R.id.nav_host_fragment_activity)

        val appBarConfiguration = AppBarConfiguration(
            setOf(
                R.id.navigation_patients, R.id.navigation_home, R.id.navigation_bluetooth
            )
        )
        setupActionBarWithNavController(navController, appBarConfiguration)
        navView.setupWithNavController(navController)
    }
}